specification/cognitiveservices/data-plane/Face/stable/v1.0/Face.json

```diff
@@ -3506,6 +3511,22 @@
           "maxLength": 16384
         }
       }
+    },
+    "MetaDataContract": {
+      "type": "object",
+      "description": "A combination of user defined name and user specified data and recognition model name for largePersonGroup/personGroup, and largeFaceList/faceList.",
+      "properties": {
+        "RecognitionModel": {
```

Comment:
We can try to extract `RecognitionModel` out as a class in definitions, to be shared inside `MetaDataContract` and `DetectedFace`.

Reply:
Done.